packages/core/src/Constants/routes.js

```diff
@@ -35,7 +37,7 @@ const routes = {
     cashier_acc_transfer: '/cashier/account-transfer',
     cashier_p2p: '/cashier/p2p',
     cashier_pa_transfer: '/cashier/payment-agent-transfer',
-    smarttrader: 'https://smarttrader.deriv.app',
+    smarttrader: `${getAllowedLocalStorageOrigin()}/br_manipulate-routing/`, // TODO: On merge, remove extra path.
```

Comment:
Since the auto-deployment is no longer here, can we remove this path now?

Reply:
Could we keep it until QA approves? If there are any new issues it will become painful to rewrite history and update the URLs again and again.